modules/tfc-agent-gke/variables.tf

```
@@ -132,5 +132,23 @@ variable "tfc_agent_name_prefix" {
 
 variable "tfc_agent_token" {
   type        = string
-  description = "Terraform Cloud agent token. (mark as sensitive) (TFC Organization Settings >> Agents)"
+  description = "Terraform Cloud Agent token. (mark as sensitive) (TFC Organization Settings >> Agents)"
+}
+
+variable "tfc_agent_image" {
+  type        = string
+  description = "The Terraform Cloud Agent image to use"
+  default     = "hashicorp/tfc-agent:latest"
+}
+
+variable "tfc_agent_memory_request" {
+  type        = string
+  description = "Memory request for the Terraform Cloud Agent container"
+  default     = "2"
```

Two bytes might be a bit light 😉 

```suggestion
  default     = "2Gi"
```

---

Defaults to 2Gi but explicit is better

---

oh maybe not for memory